src/components/Header/TopNav/Dropdown.js

```diff
@@ -91,15 +92,35 @@ export default class Dropdown extends PureComponent {
    * so we're detecting it on mouse down
    */
   handleMouseDown = () => {
+    if (this.hasTouch()) {
```

Comment:
This technique should be skipped if we're on a touch deviced — the onclick event is the one that needs to be handled for those cases